client/src/components/ApplicantsList.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
+import { getAllProfiles } from "../api/profiles";
+
+const ApplicantsList = () => {
+  const [profiles, setProfiles] = useState([]);
+  const fetchProfiles = async () => {
+    const response = await getAllProfiles();
+    setProfiles(response);
+  };
+
+  useEffect(() => {
+    fetchProfiles();
+  }, []);
+
+  return profiles.map(profile => {
+    return (
+      <div class="ui two column grid raised segment">
+        <div class="twelve wide column">
+          <label for="job title" class="ui white left ribbon label">
+            Job Title
+          </label>
+          <div class="ui small segment">
```

Comment:
Let's change this to a React Semantic UI component like https://react.semantic-ui.com/elements/segment/#types-segment